app.py

```diff
@@ -12,13 +11,16 @@ def index():
 
 @app.route('/video')
 def video():
+    print "Video request"
+
     FNULL = open(os.devnull, 'w')
 
-    hri_cmd = ['python', 'heart_rate_deamon.py', video_file, opencv_path]
-    opencv_ps = subprocess.Popen(hri_cmd, stdout=subprocess.PIPE, stderr=FNULL)
+    client_proc_cmd = 'python heart_rate_client.py ' + video_file  
```

Comment:
Well I guess the server could directly write the output to the ffmpeg, but that breaks the queue idea. Or maybe the os could be convinced to treat ZeroMQ socket as a pipe. I'm not 100% happy with this now, but it is also not an important issue. I'll merge.